assets/js/modules/analytics/datastore/setup.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * modules/analytics data store: setup.
```

Comment:
I think this part of the datastore would be better called `settings`. It's related to updating settings, which happens in both the settings section and the setup flow.

Reply:
Yeah, I thought that might be confused with settings-store-related components. That sounds fine to me though.